### DISCUSSION TITLE
Need to redirect “file not found” errors to home page

### DISCUSSION THREAD

**Discussion by gmoskowitzQF** [squidfunk/mkdocs-material#7731] [Q&A]
I am using mkdocs to create a documentation web site. The web site is accessed by an application via a Help button – when the user clicks the Help button, a specific topic page opens. Sometimes this topic page will have the wrong file name of will not exist yet. In these cases, the browser displays a “file not found” error. Instead of this error, I want it to display the home page (index.md or index.html). I can’t figure out how to do this. The mkdocs “redirects” plugin hasn’t worked yet; I don’t think it’s designed for what I want to do. Have you encountered this problem before and if so, how did you fix it?

**Comment by squidfunk** [comment]
This is a web server level setting, it has nothing to do with Material for MkDocs and MkDocs in particular. Tutorials:

- [nginx](https://www.digitalocean.com/community/tutorials/how-to-configure-nginx-to-use-custom-error-pages-on-ubuntu-14-04)
- [Apache](https://www.digitalocean.com/community/tutorials/how-to-create-a-custom-404-page-in-apache)
- or just Google "your web server + custom 404 page"
